pom.xml

```diff
@@ -144,6 +145,7 @@
         <google.zxing.version>3.4.0</google.zxing.version>
         <freemarker.version>2.3.32</freemarker.version>
         <sundrio.version>0.200.0</sundrio.version>
+        <net.java.dev.jna.version>5.6.0</net.java.dev.jna.version>
```

Comment:
Seems we're currently using `net.java.dev.jna:jna:jar:5.8.0` so this would be downgrading that quite a bit. Is there a specific reason for that?